.github/workflows/scorecards.yml

```diff
@@ -0,0 +1,69 @@
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: '39 7 * * 6'
```

Comment:
Any reason for choosing this particular time (it seems to be 7:39)? I assume having a different run time on different repo helps to not overload the servers. But I wonder if some comment explaining it would help to avoid someone changing it in the future?

Reply:
The time is generated by the scorecard github action; it is chosen randomly.